jq_array_parser.go

```diff
@@ -21,14 +19,10 @@ func NewJQArrayParser(expressions []string) *JQArrayParser {
 	}
 }
 
-func (p *JQArrayParser) Run(data string) ([]string, error) {
+func (p *JQArrayParser) Run(data string) []string {
 	output := make([]string, 0, len(p.programs))
 	for _, program := range p.programs {
-		response, err := program.Run(data)
-		if err != nil {
-			log.Warn().Err(err).Msgf("jq execution error from expression: %s", program.program.Program)
```

Comment:
Logging here is unwanted behaviour. The error message that can be returned is about a jq execution error - this is not something that needs to be logged. The user should be able to see on kubectl-opslevel that the things they expect to be picked up by JQ are not being picked up. 

From there they can `kubectl get deployment/whatever -o json | jq -r 'EXPRESSION'` to figure out what's wrong. Most libraries in general do not emit any logs and don't make a logging interface available.

If we really want logging, it should be configurable to the user of kubectl-opslevel by implementing a logging interface, but I don't see what use I would get out of it as an end user.